frontend/src/app/AppRoutes.tsx

```diff
@@ -59,7 +61,10 @@ const AppRoutes: React.FC = () => {
 
         <Route path="/projects/*" element={<ProjectViewRoutes />} />
 
-        <Route path="/notebookController/*" element={<NotebookController />} />
+        {isJupyterEnabled && (
```

Comment:
I think this should be default behaviour, not sure why don't we hide routes based on featureflags, am I missing something or should i log an issue? @andrewballantyne @christianvogt

Reply:
It's not a default behavior for now, so the route needs to be manually checked and mounted. I found an issue here already talking about this feature. It's a tech-debt https://github.com/opendatahub-io/odh-dashboard/issues/1122.

Reply:
Great!